test/e2e/autoscaling/horizontal_pod_autoscaling.go

```diff
@@ -65,6 +65,16 @@ var _ = framework.KubeDescribe("[HPA] Horizontal pod autoscaling (scale resource
 		})
 	})
 
+	framework.KubeDescribe("[Serial] [Slow] StatefulSet", func() {
```

Comment:
As discussed offline, i will:
(1) use `curl` to perform POST operations
(2) use dynamic client to modify the subresource
(3) remove HPA testing codes and save them for future usage